WooCommerce/Classes/ViewRelated/Products/Edit Product/EditableProductModel.swift

```diff
@@ -127,6 +127,18 @@ extension EditableProductModel: ProductFormDataModel, TaxClassRequestable {
         product.soldIndividually
     }
 
+    var downloads: [ProductDownload] {
+        product.downloads
+    }
+
+    var downloadLimit: Int64 {
+        Int64(product.downloadLimit)
+    }
+
+    var downloadExpiry: Int64 {
+        Int64(product.downloadExpiry)
```

Comment:
Like above 👍

Reply:
Here it's actually needed. Because `product.downloadLimit` in an `Int`. if I remove the casting, It throws a compilation error.

Reply:
Same here like above 👍

Reply:
Same thing applies as above. Please suggest.

Reply:
Like above, we can use `Int64` everywhere and avoid `Int`

Reply:
Explained [here](https://github.com/woocommerce/woocommerce-ios/pull/2814/files#r493746821)